.github/workflows/build-and-deploy-dev.yml

```
@@ -23,7 +23,7 @@ jobs:
   build-and-deploy:
     name: Build and Deploy
     needs: upload-package-lock-json
-    uses: clearlydefined/operations/.github/workflows/app-build-and-deploy.yml@v3.1.1
+    uses: clearlydefined/operations/.github/workflows/app-build-and-deploy.yml@3.1.2
```

```suggestion
    uses: clearlydefined/operations/.github/workflows/app-build-and-deploy.yml@v3.1.2
```

I think you need the `v` here